src/dmd/aggregate.d

```diff
@@ -522,7 +522,7 @@ extern (C++) abstract class AggregateDeclaration : ScopeDsymbol
      * paggalignsize: alignment of aggregate (updated)
      * isunion:       the aggregate is a union
      */
-    extern (D) static uint placeField(uint* nextoffset, uint memsize, uint memalignsize,
+    extern (D) static uint placeField(Dsymbol ds, uint* nextoffset, uint memsize, uint memalignsize,
```

Comment:
you are introducing a reference to the whole Dsymbol even though you are just using the location of it?

Reply:
Ahh, good catch. Thanks.

Reply:
Update: I've added the function name to diagnostics so using `Dsymbol` as parameter type nevertheless.